arenadata/run_gpbackup_tests.bash

```diff
@@ -2,10 +2,19 @@
 
 set -eox pipefail
 
+# 7x has additional tests with de_DE locale. Install the missing.
+# 6x has no such package and following commands are excessive, but it's not an error.
+yum install -y glibc-locale-source || true
+localedef -i de_DE -f UTF-8 de_DE
+
 source gpdb_src/concourse/scripts/common.bash
 install_and_configure_gpdb
 make -C gpdb_src/src/test/regress/
-make -C gpdb_src/contrib/dummy_seclabel/ install
+# dummy_seclabel has different installation path for 6x and 7x. Try both.
+if ! make -C gpdb_src/contrib/dummy_seclabel/ install
```

Comment:
make errors == not exist error?
may be we can add (to dockerfile) env GP_MAJOR=6 for 6X and 7 for 7X (and 5 for 5X)
and then use it
```sh
if [ "$GP_MAJOR" -eq "6" ]; then
# code for 6X
elif [ "$GP_MAJOR" -eq "7" ]; then
# code for 7X
fi
```
or
```Dockerfile
ARG GP_MAJOR=6
FROM hub.adsw.io/library/gpdb${GP_MAJOR}_regress:latest
```

Reply:
Error during first make means that we should try another directory. This works because we assume that one dummy_seclabel can't be placed in two dirs. If error is caused by the bug in dummy_seclabel itself, then you'll see it in log anyway - the second command will fail.

The ARG is a Dockerfile build command. To pass it to container you need additional instructions with ENV.
Another disadvantage of passing only the number is that you can't use any image from our hub. For example, if you want to test gpbackup with random image from our CI pipelines.
The idea of having a variable is good overall, but I found it fat. Another nit is that we should install package depends on OS version (e.g. base image of gpdb image), but not on gpdb version.
I preferred the solution which is small as possible.